alpha/template/semver/semver.go

```diff
@@ -15,12 +15,55 @@ import (
 
 	"github.com/operator-framework/operator-registry/alpha/declcfg"
 	"github.com/operator-framework/operator-registry/alpha/property"
+	"github.com/operator-framework/operator-registry/alpha/template"
 )
 
-func (t Template) Render(ctx context.Context) (*declcfg.DeclarativeConfig, error) {
+// IO structs -- BEGIN
+type semverTemplateBundleEntry struct {
```

Comment:
given that we are in `semver` package, I would suggest to drop `semver` prefix from the types declared here, the redundancy does not bring much.